interni/verbali/VI_2021_05_17/res/contenuto.tex

```diff
@@ -0,0 +1,50 @@
+\section*{Introduzione}
+\subsection*{Luogo e data dell'incontro}
+\begin{itemize}
+	\item \textbf{luogo:} videoconferenza su \glock{Discord};
+	\item \textbf{data:} 2021-05-17;
+	\item \textbf{ora di inizio:} 14:15;
+	\item \textbf{ora di fine:} 15:30.
+\end{itemize}
+
+\subsection*{Ordine del giorno}
+\begin{enumerate}
+	\item Aggiornamento sulla situazione attuale del prodotto;
+	\item Pianificazione obiettivi periodo RQ-RA;
+	\item Comunicazione con il proponente.
+		
+\end{enumerate}
+
+\subsection*{Presenze}
+\begin{itemize}
+	\item \textbf{Presenti: }
+	\begin{itemize}
+		\item Badan Antonio;
+		\item Bertoldo Damiano;
+		\item Budai Matteo;
+		\item De Grandi Samuele;
+		\item Piacere Ivan;
+		\item Privitera Sara;
+		\item Spigolon Daniele;
+	\end{itemize}
+\end{itemize}
+
+
+\section*{Svolgimento}
+
+\subsection*{Aggiornamento sulla situazione attuale del prodotto}
+Ci siamo aggiornati per quanto riguarda la situazione delle tre componenti principali del prodotto, ovvero:
+\begin{itemize}
+	\item applicazione web;
+	\item applicazione mobile;
+	\item backend.
+\end{itemize}
+Tutte e tre soddisfano i requisiti obbligatori con una percentuale superiore al 50 \%.
+Per la prima si è deciso di implementare il prima possibile la funzionalità della visualizzazione delle postazioni e stanze in modo schematico attraverso una vista a griglia, di colorare le postazioni in base allo stato in cui si trovano e infine mostrare il numero di occupanti per stanza.
+Per la seconda si è deciso di continuare con l'azione di refactoring del codice seguendo il pattern MVP e di implementare la funzionalità di visualizzazione della guida utente.
+Per la terza si è deciso di ultimare la stesura di unit test e l'implementazione della blockchain con web3 su Docker.
```

Comment:
unit test e l'implementazione della blockchain con web3 su Docker. ->
unit test, installare la blockchain in un container Docker e completare la comunicazione con quest'ultima tramite web3.